js/client.js

```diff
@@ -2472,7 +2472,8 @@
 			buf += '<strong><a href="//pokemonshowdown.com/users/' + userid + '" target="_blank">' + BattleLog.escapeHTML(name) + '</a></strong><br />';
 			var offline = data.rooms === false;
 			if (data.status || offline) {
-				buf += '<span class="userstatus' + (offline ? ' offline' : '') + '">' + (offline ? '(Offline)' : BattleLog.escapeHTML(data.status)) + '</span><br />';
+				var status = offline ? '(Offline)' : data.status.startsWith('!') ? data.status.slice(1) : data.status;
+				buf += '<span class="userstatus' + (offline ? ' offline' : '') + '">' + BattleLog.escapeHTML(status) + '</span><br />';
```

Comment:
I guess this is the main thing you'd like to merge? Sure, you should probably give your PR a new name, though.

Reply:
I don't really have much idea what this does, though.

Reply:
I mean, once you pull out the other parts of my PR and commit them separately, yeah :). I dont know if the change in `client-topbar.js` is strictly required, but the Chrome inspect seems to like it better.  The material change in this file is effectively

`data.status` -> `data.status.startsWith('!') ? data.status.slice(1) : data.status`

ie/ 'Don't include the `!` in away status display'.